packages/arb-token-bridge-ui/src/hooks/useTransactions.ts

```diff
@@ -99,7 +100,9 @@ type TransactionBase = {
   isClassic?: boolean
 }
 
-export interface Transaction extends TransactionBase {
+export interface Transaction
+  extends TransactionBase,
+    AdditionalTransferProperties {
```

Comment:
Can we just add these fields directly to `Transaction`?

Reply:
done